pcsx2/gui/AppMain.cpp

```diff
@@ -50,6 +50,18 @@
 #include <gtk/gtk.h>
 #endif
 
+#ifdef __WXMAC__
+// Great joy....
```

Comment:
I have a branch (PR #1000) to remove all those define. Could you add a comment for the future that it will be safe to remove the code.